microcosm_pubsub/tests/test_result.py

```diff
@@ -204,12 +204,13 @@ def handler(message):
         ).resolve(self.message)
 
         result.extra = dict(
+            media_type="foo.created",
```

Comment:
Not sure this tests the change above?

Reply:
It tests that the merge also works for media type.